food-truck-api/src/main/java/food/truck/api/routes/Route.java

```diff
@@ -35,4 +36,30 @@
 
     @ElementCollection
     Set<DayOfWeek> days = EnumSet.noneOf(DayOfWeek.class);
+
+    @Override
+    public String toString() {
```

Comment:
I'm confused about these changes. The Data annotation generates all three of these methods for you behind the scenes.

Reply:
I think this is because "days" and "locations" were lazily loaded, which caused things to break with a transaction error in weird places if a route got converted to a string